samples/sensor/ccs811/boards/nrf51_ble400.overlay

```diff
@@ -5,17 +5,17 @@
  */
 
 &i2c0 {
-        status = "okay";
-        sda-pin = <0>;
-        scl-pin = <1>;
+	status = "okay";
+	sda-gpios = <&gpio0 0 0>;
+	scl-gpios = <&gpio1 1 0>;
```

Comment:
```suggestion
	scl-gpios = <&gpio0 1 0>;
```

Reply:
hah, the one change I don't do with a script ... :)